src/ShallowWrapper.js

```diff
@@ -350,8 +350,11 @@ export default class ShallowWrapper {
     const handler = this.prop(propFromEvent(event));
     if (handler) {
       withSetStateAllowed(() => {
-        // TODO(lmr): create/use synthetic events
         // TODO(lmr): emulate React's event propagation
+        if (!args.length && window) {
+          args.push(new window.Event(event));
```

Comment:
👍 So what would be better approach ? Check for `global.window` and add it only for users that have it exposed ? (Leads to inconsistency) 
Or create manually event object ? (Don't think so we can mock properly all properties)